### DISCUSSION TITLE
HA update to 2023.8.x

### DISCUSSION THREAD

**Discussion by rizydorczyk** [tteck/Proxmox#1684] [Help Needed]
Hi.

I can't update HA Core with script. It was in the version 2023.7.3

iZYD

**Comment by tteck** [comment]
Try executing the command below in the LXC console

```
systemctl stop homeassistant.service && source /srv/homeassistant/bin/activate && pip3 install --upgrade homeassistant && systemctl start homeassistant.service && exit
```

**Reply by rizydorczyk** [reply]
from console 

`[notice] To update, run: pip install --upgrade pip`

**Comment by tteck** [comment]
What does the LXC console display when you run the command `python3 -V`?

**Reply by rizydorczyk** [reply]
root@homeassistant-core:~# python3 -V
Python 3.10.8

**Reply by tteck** [reply]
Python 3.10 support has previously been deprecated and has now been removed. Home Assistant now requires Python 3.11 to run.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/issues/1469

**Reply by rizydorczyk** [reply]
thx moved to new lxc
